Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -520,6 +520,8 @@
       - MassMediaCircuitboard
       - ReagentGrinderIndustrialMachineCircuitboard
       - JukeboxCircuitBoard
+      - SMESAdvancedMachineCircuitboard
+
```

Comment:
None of the other lathes have spacing between these lines, and it can cause confusion if this is left here.

```suggestion
```